### ISSUE TITLE
Great Job! I want to know which parts of the code were mainly modified in the implementation of this UCAS module?

### ISSUE THREAD

**Issue by JiangYubo4399** [xvolcano02/UCAS#2]

**Comment by xvolcano02** [comment]
@JiangYubo4399 Hi! Thanks for the question!
The UCAS implementation mainly modified the following parts:
### Core UCAS algorithm logic
Added/updated uncertainty-aware advantage shaping computation (the core UCAS behavior in the [verl.trainer.ppo.core_algos.py](https://github.com/xvolcano02/UCAS/blob/main/verl/trainer/ppo/core_algos.py)).
### Actor path (new logits trace added)
Added a new logits trace in the actor to record model logits along generation( in the [verl.trainer.workers.actor.dp_actor.py).](https://github.com/xvolcano02/UCAS/blob/main/verl/workers/actor/dp_actor.py)
This trace is used by UCAS-related computation/analysis and is a key new addition.
### Training / optimization pipeline
Integrated UCAS signals into the RLVR training loop.
Updated how advantage/loss terms are computed and applied.
